code/datums/votes/map_vote.dm

```diff
@@ -39,11 +39,14 @@
 	if(. != VOTE_AVAILABLE)
 		return .
 
+	if(SSmap_vote.next_map_config)
+		return "The next map has already been selected."
+
+	default_choices = SSmap_vote.get_valid_map_vote_choices()
```

Comment:
Careful, `can_be_initiated` is called in ui_data, may not be wise to check this every tick of it

Reply:
only good solution here would be caching the amount of players during last refresh, probably gonna do that?